build.go

```diff
@@ -157,10 +179,10 @@ func buildChild(k *Kong, node *Node, typ NodeType, v reflect.Value, ft reflect.S
 	// a positional argument is provided to the child, and move it to the branching argument field.
 	if tag.Arg {
 		if len(child.Positional) == 0 {
-			failField(v, ft, "positional branch must have at least one child positional argument named %q", name)
+			return failField(v, ft, "positional branch must have at least one child positional argument named %q", name)
```

Comment:
These were originally intended to be actually surfaced as panics, not errors (as they are unrecoverable programmer errors), but I think that must have changed along the way. As they're already errors, this seems reasonable.